modules/core/include/opencv2/core/utils/logger.configuration.hpp

```diff
@@ -0,0 +1,28 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef OPENCV_LOGGER_CONFIGURATION_HPP
+#define OPENCV_LOGGER_CONFIGURATION_HPP
+
+//! @addtogroup core_logging
+//! @{
+
+namespace cv {
+namespace utils {
+namespace logging {
+
+// Supported logging levels and their semantic
+#define CV_LOG_LEVEL_SILENT 0          //!< for using in setLogLevel() call
+#define CV_LOG_LEVEL_FATAL 1           //!< Fatal (critical) error (unrecoverable internal error)
+#define CV_LOG_LEVEL_ERROR 2           //!< Error message
+#define CV_LOG_LEVEL_WARN 3            //!< Warning message
+#define CV_LOG_LEVEL_INFO 4            //!< Info message
+#define CV_LOG_LEVEL_DEBUG 5           //!< Debug message. Disabled in the "Release" build.
+#define CV_LOG_LEVEL_VERBOSE 6         //!< Verbose (trace) messages. Requires verbosity level. Disabled in the "Release" build.
+
```

Comment:
I think `CV_LOG_STRIP_LEVEL` macro definition can be moved here.

Reply:
See updated PR description.

Reply:
It should work even if we define _level_ macros after _strip_ macro:
```.cpp
#define CV_LOG_STRIP_LEVEL CV_LOG_LEVEL_DEBUG
#include <opencv2/core/utils/logger.hpp>
```